### ISSUE TITLE
Permissions error when using https to access github

### ISSUE THREAD

**Issue by ttmarek** [rangle/rangle-starter#134]
I got the error below when trying to set up a react+redux project using the `rangle-starter` cli.
A few notes:
- I selected https as the protocol for connecting to github, but it looks like `rangle-starter` is trying to clone the `react-redux-starter` repo using its ssh url.

`````` sh
➜  Desktop rangle-starter
? Please choose a tech stack. React/ES6/Redux
? Please enter the name of your new project play-react-redux-starter
? Would you like to use up a Github repo with this project ? Yes
? Please chose a protocol for connecting to github https: ( https://github.com/y
ourOrg/yourRep )
? Please enter the git account that this project will belong to ttmarek
? Please enter your git account if you'd like to use a personal fork
( leave empty to skip ):
git clone git@github.com:rangle/react-redux-starter --depth=1 play-react-redux-starter
Cloning into 'play-react-redux-starter'...

Warning: Permanently added the RSA host key for IP address '192.30.252.123' to the list of known hosts.

Permission denied (publickey).

fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.

Error: git exited with code 128
    at ChildProcess.child.on.code (/Users/marek/.nvm/versions/node/v6.2.2/lib/node_modules/rangle-starter/src/execute.js:73:16)
    at emitTwo (events.js:106:13)
    at ChildProcess.emit (events.js:191:7)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:204:12)```
``````

**Comment by ttmarek** [comment]
I went in to fix this myself but noticed that it was already fixed by 9bcb1ca05b7749064a1cc1b3ea816ec18f6e9eca. This issue can be closed.

**Comment by SethDavenport** [comment]
Published 1.3.1 to NPM with this fix.

**Comment by SethDavenport** [comment]
@ttmarek can you verify and close if it's working for you now?

You'll need to do another `npm install -g rangle-starter`
